robotbase/src/main/java/com/team2357/frc2022/shuffleboard/AutoModeCommandChooser.java

```diff
@@ -0,0 +1,93 @@
+package com.team2357.frc2022.shuffleboard;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+// TODO: ShuffleBoard Implement
+/**
+ * This class should control a chooser to select up to three actions with a wait
+ * command between them
+ * Do not worry about making the actual auto commands, just the chooser widget
+ * that can select different
+ * things is enough for now
+ */
+public class AutoModeCommandChooser {
+    public enum AutomodeActions {
+        NONE,
+        MOVE_OFF_LINE,
```

Comment:
```NONE``` is the only thing we know right now we will have right now. This means ```MOVE_OFF_LINE``` and ```SHOOT``` can go away.